doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -504,6 +504,13 @@ Matter
 
 The issues in this section are related to the :ref:`ug_matter` protocol.
 
+.. rst-class:: v2-9-0 v2-8-0 v2-7-0
```

Comment:
the issues with certification testing will be since Matter 1.2.
the first NCS release whit Matter 1.2 was NCS 2.6.0.
So i suggest to add 
v2.6.0 v2.6.1 v2.6.2

Reply:
added